### ISSUE TITLE
Correctness of viewshed analysis

### ISSUE THREAD

**Issue by yiftachn** [makepath/xarray-spatial#669]
Has anyone tested the correctness of the viewshed analysis output?
Assuming ESRI's ArcGIS as the ground truth, as it is the most popular, paid, solid and established viewshed product, 
Does xarray-spatial viewshed returns identical output?

**Comment by ianthomas23** [comment]
I don't think you will find must interest here in comparing results with commercial closed-source software. Personally I don't trust any mathematical software unless I can verify that the maths is implemented correctly.

It you want to compare the two, feel free to do so and report your results here if you wish.

**Comment by yiftachn** [comment]
Great, im following
